ui/README.md

```diff
@@ -27,10 +29,32 @@ Before committing, be sure to run `make` to generate a non-debug version of
 will result in wasted time waiting for the build.
 
 ## Dependencies
-Our admin UI is written in Typescript, so compiling it depends on having the
-typescript compiler locally. We also require a typescript linter (tslint).
+Our admin UI is compiled using a collection of tools that depends on
+[nodejs](https://nodejs.org/), so you'll want to have that installed.
```

Comment:
I don't think so; this is just saying "hey, you'll need node installed or things won't work"